internal/cmd/release/main.go

```diff
@@ -504,30 +502,16 @@ func createPluginReleases(dir string, plugins []release.PluginRelease) error {
 	return encoder.Encode(&release.PluginReleases{Releases: plugins})
 }
 
-func pullImage(name string) error {
-	cmd, err := dockerCmd("pull", name)
-	if err != nil {
-		return err
-	}
+func pullImage(ctx context.Context, name string) error {
 	log.Printf("pulling image: %s", name)
-	return cmd.Run()
+	return dockerCmd(ctx, "pull", name).Run()
 }
 
-func dockerCmd(command string, args ...string) (*exec.Cmd, error) {
-	dockerPath, err := exec.LookPath("docker")
-	if err != nil {
-		return nil, err
-	}
-	cmd := &exec.Cmd{
-		Path: dockerPath,
-		Args: append([]string{
-			dockerPath,
-			command,
-		}, args...),
-		Stdout: os.Stdout,
-		Stderr: os.Stderr,
-	}
-	return cmd, nil
+func dockerCmd(ctx context.Context, command string, args ...string) *exec.Cmd {
+	cmd := exec.CommandContext(ctx, "docker", append([]string{command}, args...)...) //nolint:gosec
```

Comment:
I'm assuming all these gosec lints are just due to possibly `exec`ing user-given args, which we're not concerned about?

Reply:
Yeah exactly. These are internal commands used by the build process.